forbiditerative/plan.py

```diff
@@ -281,7 +277,8 @@ def validate_input(args):
         title="time and memory limits")
     lim.add_argument("--overall-time-limit")
     lim.add_argument("--overall-memory-limit")
- 
+
+    parser.add_argument("--build", help="absolute path for planner binaries")
```

Comment:
--build arg is used in fast_downward to provide the build config name (default is "release"), please see the file build_configs.py for details.